Exercise-A-Static-page/user.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise A</title>
+</head>
+
+<body>
+    <h1>Exercise A</h1>
+    Id: 1
+    Name: Sarah
+    Adress: Entença 95
```

Comment:
Could you put lastname and forename please?

Reply:
It's done. Thank You.

Reply:
Hey! Great
Last think I hope: can you keep the spacing around the colons consistent (sometimes blalba:blalba and sometimes blalba: blalba )
Also check what is the best practice in english, space before, after or both
thanks

Reply:
Done Thanks.